internal/hud/server/view.go

```diff
@@ -62,6 +62,11 @@ func StateToWebView(s store.EngineState) webview.View {
 
 		endpoints := store.ManifestTargetEndpoints(mt)
 
+		var podIDs []string
+		for id, _ := range ms.PodSet.Pods {
```

Comment:
in other places in the UI, we've used GetMostRecentPod() to show the most relevant pod, rather than just showing all of them

Reply:
Huh, this entire file seems to be gone now...